repository/l.json

```diff
@@ -1147,6 +1147,17 @@
 				}
 			]
 		},
+		{
+			"name": "LionDocs",
+			"details": "https://github.com/liondocs/liondocs-sublime",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": "*",
```

Comment:
`"*"` is assumed, so you don't need to specify "platforms" here.

Reply:
Removed